src-ghc/Pact/Bench.hs

```diff
@@ -23,10 +24,7 @@ import qualified Data.HashMap.Strict as HM
 import qualified Data.Map.Strict as M
 import Data.Text (unpack, pack, intercalate)
 import Data.Text.Encoding
-
--- import GHC.Clock
-import Data.Thyme.Clock
-import Data.AffineSpace
+import "pact-time" Data.Time
```

Comment:
Please don't re-use `Data.Time` if there's a conflict, use `Pact.Time` etc. Package imports should only be used when you don't control the dep code.